src/script/interpreter.cpp

```diff
@@ -1348,7 +1349,7 @@ bool TransactionSignatureChecker::CheckLockTime(const CScriptNum& nLockTime) con
     return true;
 }
 
-bool TransactionSignatureChecker::CheckBlockHash(const int32_t nHeight, const std::vector<unsigned char>& vchCompareTo) const
+bool CheckReplayProtectionData(const CChain* chain, int nHeight, const std::vector<unsigned char>& vchCompareTo)
```

Comment:
Should we call this also from bool CertificateSignatureChecker::CheckBlockHash to avoid code duplication?

Reply:
Done